### DISCUSSION TITLE
Can we turn off the vibrate in text aligning?

### DISCUSSION THREAD

**Discussion by dna2me** [hm21/pro_image_editor#297] [Q&A] [answered]
When I add a text box, I try to move it to somewhere in the screen. However it always vibrates when I touched the vertical align line. I see the code is in layer_interaction_manager.dart, but it seems not to be configurable.

Can we make it configurable? I just don't want it vibrate. The center line is enough for me.
Thanks a lot.

**Comment by hm21** [comment, marked as answer]
The vibration feature can be disabled within the `helperLines` configuration settings, as shown below:
```dart
configs: ProImageEditorConfigs(
    helperLines: const HelperLines(
      hitVibration: false,
    ),
```
